src/CalcManager/pch.h

```diff
@@ -24,6 +22,22 @@
 #include <list>
 #include <regex>
 #include <unordered_map>
-#include <intsafe.h>
 #include <array>
+#include <string_view>
+#include <future>
+
+#include "win_data_types_cross_platform.h"
+
+#if defined(_WIN32) && defined(_MSC_VER)
+
+#include <windows.h>
+#include <winerror.h>
 #include <ppltasks.h>
```

Comment:
Is `ppltasks.h` still a required include with changes to `src/CalcManager/UnitConverter.{cpp,h}`?

Reply:
Fixed by 10a9a9f